ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -91,8 +99,14 @@ function BlockaidBannerAlert({ txData, ...props }) {
     domain: origin ?? msgParams?.origin,
     jsonRpcMethod: type,
     jsonRpcParams: JSON.stringify(txParams ?? msgParams),
+    blockNumber: block,
+    chain: NETWORK_TO_NAME_MAP[chainId],
     classification: reason,
+    blockaidVersion: BlockaidPackage.version,
+    resultType,
+    reproduce: JSON.stringify(features),
```

Comment:
comment: curious how we named this reproduce. I see it is used for the URI param. why reproduce and not details or features?